parser-typechecker/src/Unison/Codebase.hs

```diff
@@ -88,15 +88,22 @@ data Codebase m v a =
            , branchHashesByPrefix :: ShortBranchHash -> m (Set Branch.Hash)
            }
 
--- | Write all of the builtins types and IO types into the codebase
+-- | Write all of the builtins types and IO types into the codebase. Returns the names of builtins
+-- but DOES NOT add these names to the namespace.
+initializeBuiltinCode :: forall m. Monad m => Codebase m Symbol Parser.Ann -> m Names.Names0
+initializeBuiltinCode c = do
+  let uf = (UF.typecheckedUnisonFile (Map.fromList Builtin.builtinDataDecls)
+                                     (Map.fromList Builtin.builtinEffectDecls)
+                                     mempty mempty)
+  addDefsToCodebase c uf
+  addDefsToCodebase c IOSource.typecheckedFile
+  pure $ Builtin.names0 <> UF.typecheckedToNames0 IOSource.typecheckedFile
```

Comment:
It might be clearer if this didn't do anything with Names?  
```haskell
-- | Write all of the builtin term and type definitions into the codebase.
-- (Doesn't do anything with namespaces.)
writeBuiltinDefinitions :: forall m. Monad m => Codebase m Symbol Parser.Ann -> m ()
```
and then move the return value back into `initializeCodebase` or into a separate top-level definition. (The value isn't derived from earlier execution in this function, so it seems random.)

Reply:
good idea, did this

Reply:
the remaining `pure ()` is redundant